test/Lint.lean

```diff
@@ -57,3 +57,23 @@ set_option linter.dupNamespace true in
 export Nat (add)
 
 end add
+
+set_option linter.cdot false in
+/--
+warning: Please, use '·' (typed as `\·`) instead of '.' as 'cdot'.
+note: this linter can be disabled with `set_option linter.cdot false`
+---
+warning: Please, use '·' (typed as `\·`) instead of '.' as 'cdot'.
+note: this linter can be disabled with `set_option linter.cdot false`
+---
+warning: Please, use '·' (typed as `\·`) instead of '.' as 'cdot'.
```

Comment:
Would you like to test a second kind of wrong dot as well, e.g. change of the wrong dots? (I was wondering if the wrong dot was hardcoded - the answer is "clearly not", but a test could be nice.)

Reply:
I added another test, with `(. + ·)`, swapping the positions of `·` and `.`.

Reply:
Thanks! I had meant something else: are there other characters which are treated like a cdot, but are also bad.
Reading `Init/NotationExtra.lean` in core, the current answer is no - so this is fine as-is.